MdeModulePkg/Universal/CapsulePei/Capsule.h

```diff
@@ -45,18 +46,18 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 

```

Comment:
Looks like this commit can be dropped? Just adding spaces and the one header, which seems like isn't needed.